modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/controls/BulkActions.js

```diff
@@ -93,8 +110,14 @@ function BulkActions({
 				loadData,
 				namespace,
 				selectedData: {
+					...(allItemsSelectedActive && {
+						filters: getActiveFilters(filters.filters) ?? {},
+					}),
 					items: allItemsSelectedActive ? [] : selectedItems,
 					keyValues: allItemsSelectedActive ? [] : selectedItemsValue,
+					...(allItemsSelectedActive && {
+						searchQuery: searchParam ?? {},
```

Comment:
The fallbacks seem incorrect, but they aren't needed since `getActiveFilters` will always return an array and `searchParam` [is initialized as an empty string already](https://github.com/liferay/liferay-portal/blob/a5ae22605c8a3d5f2ea82454a6a96cad8de2b2bb/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSet.tsx#L164).

So this can be simplified to (with the previous comment change applied):
```
filters: getActiveFilters(filters)
...
searchQuery: searchParam
```